task/06-conditions-n-loops-tasks.js

```diff
@@ -150,7 +178,11 @@ function doRectanglesOverlap(rect1, rect2) {
  *   
  */
 function isInsideCircle(circle, point) {
-    throw new Error('Not implemented');
+    let distanceToCenter = Math.sqrt(Math.pow(point.x - circle.center.x, 2) + Math.pow(point.y - circle.center.y, 2));
+    if (distanceToCenter < circle.radius) {
+        return true;
+    }
+    return false;
```

Comment:
Please, try to solve just using `return ... ` . Without if, without true or false